superset-frontend/cypress-base/cypress/integration/dashboard/nativeFilter.helper.ts

```diff
@@ -0,0 +1,412 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { dashboardView, nativeFilters } from 'cypress/support/directories';
+import { testItems } from './dashboard.helper';
+
+export const nativeFilterTooltips = {
+  searchAllFilterOptions:
+    'By default, each filter loads at most 1000 choices at the initial page load. Check this box if you have more than 1000 filter values and want to enable dynamically searching that loads filter values as users type (may add stress to your database).',
+  defaultToFirstItem: 'When using this option, default value can’t be set',
+  inverseSelection: 'Exclude selected values',
+  required: 'User must select a value before applying the filter',
+  multipleSelect: 'Allow selecting multiple values',
+  defaultValue:
+    'Default value must be set when "Filter value is required" is checked',
+};
```

Comment:
I am considering the constants whether uses the _original text_ of components rather than hard code in the test case. How about you see this? @michael-s-molina

Reply:
I think using constants is a good idea but that would require changing the original components to also use these constants right?

Reply:
The original components and test case might share common constants if needed

Reply:
can i do this in a separated PR  when it is ready?